include/openssl/ssl.h

```diff
@@ -1899,6 +1899,15 @@ int DTLSv1_listen(SSL *s, BIO_ADDR *client);
 # ifndef OPENSSL_NO_CT
 
 /*
+ * A callback for verifying that the received SCTs are sufficient.
+ * Expected to return 1 if they are sufficient, otherwise 0.
+ * May return a negative integer if an error occurs.
```

Comment:
So maybe instead, define new named constants `OPENSSL_CT_OK` (= 1) and `OPENSSL_CT_NOACK`, and in code check for `== OPENSSL_CT_OK`.